code/datums/diseases/decloning.dm

```diff
@@ -35,18 +38,17 @@
 			if(SPT_PROB(1, seconds_per_tick))
 				affected_mob.emote("drool")
 			if(SPT_PROB(1.5, seconds_per_tick))
-				affected_mob.adjustCloneLoss(1, FALSE)
+				affected_mob.apply_status_effect(/datum/status_effect/decloning)
 			if(SPT_PROB(1, seconds_per_tick))
 				to_chat(affected_mob, span_danger("Your skin feels strange."))
-
 		if(4)
 			if(SPT_PROB(1, seconds_per_tick))
 				affected_mob.emote("itch")
 			if(SPT_PROB(1, seconds_per_tick))
 				affected_mob.emote("drool")
 			if(SPT_PROB(2.5, seconds_per_tick))
+				affected_mob.apply_status_effect(/datum/status_effect/decloning)
```

Comment:
the status effect won't stack up multiple times on the same person if these fire, right?